nmrfx-analyst-gui/src/main/java/org/nmrfx/processor/gui/GUIScripter.java

```diff
@@ -105,7 +105,8 @@ public boolean active(int index) throws IllegalArgumentException {
         if (useChart != null) {
             return false;
         }
-        ConsoleUtil.runOnFxThread(() -> {
+        // fixme throwing from fx thread
+        Fx.runOnFxThread(() -> {
```

Comment:
to be fixed in future?

Reply:
That's an old comment that was moved during copy/paste it seems.